specs/sb/api.md

```diff
@@ -573,9 +573,9 @@ existing system.
 </thead>
 <tbody>
 <tr>
-  <td>organization_guid*</td>
+  <td>context_guid*</td>
```

Comment:
I'd really prefer to put the burden on the platform to manage this. And, in fact, I would think it would be a bit of a security issue if the only source of truth for the space was the SB because then what if it lied? Wouldn't it be safer for the platform to always just send a GUID (which in some cases could be a SHA/HASH of the info) so that the SB couldn't muck with the info at all?